src/SqlPersistence/Saga/RuntimeSagaInfo.cs

```diff
@@ -62,9 +66,23 @@ class RuntimeSagaInfo
         {
             case SqlVariant.MsSqlServer:
                 TableName = $"[{schema}].[{tablePrefix}{tableSuffix}]";
+                FillParameter = ParameterFiller.Fill;
                 break;
             case SqlVariant.MySql:
                 TableName = $"`{tablePrefix}{tableSuffix}`";
+                FillParameter = ParameterFiller.Fill;
+                break;
+            case SqlVariant.Oracle:
+                if (tableSuffix.Length > 27)
+                {
+                    throw new Exception($"Saga '{tableSuffix}' contains more than 27 characters, which is not supported by SQL persistence using Oracle. Either disable Oracle script generation using the SqlPersistenceSettings assembly attribute, shorten the name of the saga, or specify an alternate table name using the SqlSagaAttribute's 'tablePrefix' parameter.");
```

Comment:
This can't be centralized because it's needed both in SqlPersistence and ScriptBuilder and they have no common dependency. I agree conceptually though.